src/lib/coda_lib/coda_lib.ml

```diff
@@ -421,7 +421,7 @@ let create (config : Config.t) =
                     (Some persistence, root_snarked_ledger, frontier) )
           in
           let frontier_broadcast_pipe_r, frontier_broadcast_pipe_w =
-            Broadcast_pipe.create (Some transition_frontier)
+            Broadcast_pipe.create None
```

Comment:
I'm not totally sure it makes sense to build the initial transition frontier in Coda_lib anymore. Shouldn't the creation of the transition frontier be tucked underneath Transition_router now?

Reply:
I think we should still keep it there, because in the future after persistence stuff gets fixed, we still need to first load the frontier from the desk. I don't think this logic belongs to transition_router.